app/cards/Card.tsx

```diff
@@ -0,0 +1,68 @@
+import * as marked from 'marked';
+import * as React from 'react';
+import { gql, graphql } from 'react-apollo';
+import { Card as ICard } from '../../server/models/';
+
+interface Props {
+  data: {
+    card: ICard;
+    networkStatus: number;
+  };
+  match: {
+    params: {
+      id: string;
+    };
+  };
+}
+
+const cardDataFromId = gql`
+  query getCardById($id: String!) {
+    card(id: $id) {
+      id
+      title
+      authors {
+        name
+      }
+      created
+      updates
+      content
+    }
+  }
+`;
+
+const config = {
+  options: ({ match }) => ({ variables: { id: match.params.id } }),
```

Comment:
`match` is provided by react router.

`options` is a function which receives the `props` object at runtime and returns an object containing the variables needed to run the query. In this case, we're using `props.match`, or `{ match }` as a shorthand and setting that value of `props.match.params.id` to `id` in the graphql query.